library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
Hey guys would it be best to come up with a list of always used tools and in the future look at a way of including them if they are needed. Not sure how to do it yet , haven't thought it thru but if there is a call for bootstrap them make bootstrap available, this is computer programming there has to be a way. It is just like "There would have never been a fin on a fishes back if there was not a water for him to swim in"